spec/classes/bird_spec.rb

```diff
@@ -19,6 +27,8 @@
           }
         end
 
+        it { is_expected.not_to contain_yumrepo('bird') }
+        it { is_expected.not_to contain_package('bird').with_require('Yumrepo[bird]') }
```

Comment:
Better to use the[ relationship matcher](https://github.com/rodjek/rspec-puppet#relationship-matchers)?